package.json

```diff
@@ -15,6 +15,7 @@
     "escodegen": "^1.12.0",
     "eslint": "^6.4.0",
     "eslint-config-prettier": "^6.3.0",
+    "eslint-config-react-app": "^5.0.2",
```

Comment:
Is it necessary to add this "globally", or would it suffice to add it to the example-specific page `package.json` file?
(If so, the `package-lock.json` file can be reverted as well.)

Reply:
It was needed before i add typescript to the project, after that it's really not necessary, already removed it

Reply:
I remembred the reason of the config, it's because of the Ci failure, should i add it again?

![image](https://user-images.githubusercontent.com/19766035/66491408-0e216700-ea89-11e9-999a-f0b3846c30f2.png)

Reply:
Would it work to add it to `examples/create-react-app/package.json` file, rather than the top-level `package.json` file?

Reply:
Didn't work, i tried it, only in the top-level :(

Reply:
I also don't think we should have this here because the example should be self-contained. If need be, we can exclude the directory from ESLint, but I would suggest trying to remove the following lines from your `package.json` first:

```
  "eslintConfig": {
    "extends": "react-app"
  },
```

Perhaps that works?

Reply:
i'll try it

Reply:
Worked! Thanks @timvandermeij!